doc/source/development/code_style.rst

```diff
@@ -18,7 +18,7 @@ consistent code format throughout the project. For details see the
 Patterns
 ========
 
-foo.__class__
+Foo.class
```

Comment:
Actually this is on purpose, does ```validate_rst_title_capitalization.py``` showing this as an error?

Reply:
It is showing this as an error, because foo is not inside exceptions.
But moreover, it is removing underscores from the title here:
`with open(rst_file, "r") as fd:
        previous_line = ""
        for i, line in enumerate(fd):
            line = line[:-1]
            line_chars = set(line)
            if (
                len(line_chars) == 1
                and line_chars.pop() in symbols
                and len(line) == len(previous_line)
            ):
                **yield re.sub(r"[{backtick}\*_]", "", previous_line)**, i
            previous_line = line`

Reply:
@cleconte987 can you change the title to `Using foo.__class__` instead? And below same for `Using f-strings`.

This will make the validation pass, and things look probably even better, without adding extra complexity.

Thanks!

Reply:
Roger that!